bundles/org.openhab.binding.dominoswiss/README.md

```diff
@@ -0,0 +1,110 @@
+
+```
+# Configuration for the Philips Hue Binding
+#
+# Default secret key for the pairing of the Philips Hue Bridge.
+# It has to be between 10-40 (alphanumeric) characters
+# This may be changed by the user for security reasons.
+secret=openHABSecret
+```
+
+_Note that it is planned to generate some part of this based on the information that is available within ```src/main/resources/OH-INF/binding``` of your binding._
+
+_If your binding does not offer any generic configurations, you can remove this section completely._
+
+## Thing Configuration
+
+The bridge "egate" has one channel "getconfig" which returns the config of this bridge. 
+The egate is configured with both an ipaddress and a port. The ip address might be in format like 192.168.1.1 or as the hostname representing the egate in your network.  Port is represented as a number.`
+
+```java
+Bridge dominoswiss:egate:myEgate "My EGate Server" @ "Home" [ ipAddres="localhost", port="5700" ]
+```
+
+
+The thing blind represents one blind on the egate. Each blind is represented by an id set on your egate.
+  
+```java 
+Thing blind officeBlind "Office" @ "1stFloor" [ id="1"]
+```
+
+The blind-Thing has the following channels:
+| Channel Type ID  | Item Type | Description                                                                                       |   |   |
+|------------------|-----------|---------------------------------------------------------------------------------------------------|---|---|
+| pulseup            |Blind      |sends one pulse up to this blind.|
+| pulsedown          |Blind      |sends one pulse down to this blind|
+| continuousup       |Blind      |sends a continuous up to this blind. The blind will automatically stop as it is fully up.|
+|continuousdown      |Blind      | send a continous down to this blind. The blind will automatically stop as it is fully down.|
+| stop               |Blind      | stop the action of the blind. The command will be imadiatly sent to the blind.|
+| shutter            |Blind      |this is used to bind the channel to the shutter item. There are no further rules needed this channel will handel the up/down/stop commands. See example for usage.|
+| tilt               |Blind      | same as shutter, this will handel all up/down/stop - tilt commands to be used with the shutter-item.| 
+| tiltup             |Blind      | sends 3 pulse-up commands to this blind to tilt the blind. If your blind needs more than 3 pulse-up, create a rule yourself with three pluse-up commands. Between each pulse-up you should wait 150ms to let the command be processed. 
+| tiltdown           |Blind      |sends 3 pulse-down commands to this blind to tilt the blind. If your blind needs more than 3 pulse-down, create a rule yourself with three pluse-down commands. Between each pulse-down you should wait 150ms to let the command be processed. |
```

Comment:
The channels should use the Java naming called lowerCamelCase. See....

https://www.openhab.org/docs/developer/guidelines.html#java-coding-style

Reply:
Thank you @Skinah for your input, I fixed that in my new commit